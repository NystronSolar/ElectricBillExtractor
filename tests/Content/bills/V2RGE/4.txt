TIMOTHY DA SILVA
R FICTICIA 123 
CENTRO
12345-678 CIDADE RS Nota Fiscal 
Conta de Energia Elétrica
Nº 012345678 Série U
Data de Emissão: 09/12/2021
Data de Apresentação:  14/12/2021
Pág: 01 de 01
Conta Contrato N° 012345678901
Leitura Próximo Mês:  07/01/2022
Lote Roteiro de leitura N°. Medidor PN
17 ABCDE012-00000123 0123456 012345678 Reservado ao Fisco
0000.0000.0000.0000.0000.0000.0 000.0000
Mantenha seus dados sempre  atualizados, alguns itens  determinam a tarifa e  tributação de sua fatura de  energia elétrica. Solicite os  serviços disponíveis em nosso  site com rapidez
e segurança e reserve mais  tempo para você em seu  dia-a-dia. Mais informações  acesse o endereço que consta no  verso de sua conta.
TIMOTHY DA SILVA                                 
R FICTICIA 123 
CENTRO
12345-678 CIDADE - RS RG: 0123456789
CLASSIFICAÇÃO: Convencional B1  Residencial - Bifásico 220 /  127 V
0800 970 0900
www.rge-rs.com.br 716097159 INSTALAÇÃO
0123456789 DEZ/2021 21/12/2021 **********
Cod.
012 Descrição da Operação
N° 012345678901 Mês
Ref. Quant.
Faturada Unid.
Med. Tarifa com
Tributos R$ Valor Total
da Operação
R$ Base Cálculo
ICMS R$ Aliq.
ICMS ICMS Base Cálculo
PIS/COFINS PIS
0,85% COFINS
3,99%
0605 Custo Disp Uso Sistema TUSD DEZ/21 50,000 kWh 0,42800000  21,40  21,40  12,00 2,57  18,83  0,16  0,75 
0601 Disp Sistema-TE DEZ/21 50,000 kWh 0,33980000  16,99  16,99  12,00 2,04  14,95  0,13  0,60 
0601 Adicional D Sist Bandeira DEZ/21 50,000 kWh 0,16960000  8,48  8,48  12,00 1,02  7,46  0,06  0,30 
0804 Juros de Mora NOV/21 0,35 
0805 Multa por Atraso Pgto NOV/21 7,24 
0805 Atualização Monetária IPCA NOV/21 0,45 
Total Distribuidora 54,91 
DÉBITOS DE OUTROS SERVIÇOS
0807 Contrib. Custeio IP-CIP  Municipal DEZ/21 1,17 
       
       
       
       
       
Total Consolidado             56,08  46,87  5,63  41,24  0,35  1,65  Bandeiras
Tarifárias
(Dias)
Esc Hídrica
22 Dias
Esc Hídrica
09 Dias
             
2021 DEZ lll
20 31
     NOV lllllllllllllllllllllllllllllll llllllllllllllllllll
304 33
     OUT lllllllllllllllllllllllllllllll lllllllll
241 28
     SET lllllllllllllllllllllllllllllll lllllllllllllllllllllllllllll
362 33
     AGO lllllllllllllllllllllllllllllll lllllllllllllllllllllllllllllll lllllllllllll
449 31 Consumo TUSD           TE
Consumo kWh 0,35850000     0,28459000
                                              
                                              
                                              
                                              
                                              
                                              
                                              
                                               Nº Energia Leitura Leitura Fator Consumo Taxa de Perda Leitura
 09/12/2021 08/11/2021 Multipl. [kWh] [%] Próximo Mês
01234567 Ativa 32202 32182   1,00       20 07/01/2022
Para consulta dos indicadores  acesse nosso site  www.rge-rs.com.br
Excepcionalmente este mês  estamos postergando sua data de  vcto.,
de modo a atender os prazos da  Resolução 012/ANEEL/0123.
Não Pague. Para sua comodidade  o valor devido R$ 56,08 será
cobrado, sem acréscimo em conta  futura. Caso não queira
esta opção, é possível  solicitar o código de barras
para pagamento e o cancelamento  desse serviço através de nossos canais de  atendimento.
CódDébAut-Banco
012345678901Nota Fiscal
Conta de Energia Elétrica
N° 0123456 Série U Total a Pagar (R$)
********** Data de Vencimento
21/12/2021
NÃO RECEBER. O VALOR DESTA  FATURA SERÁ INSERIDO EM CONTA  FUTURA ISENTO DE ENCARGOS
SAIBA MAIS EM:  www.rge-rs.com.br/contaminima