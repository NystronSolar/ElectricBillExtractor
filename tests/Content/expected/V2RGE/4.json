{
    "client": {
        "name": "TIMOTHY DA SILVA",
        "address": {
            "street": "R FICTICIA 123",
            "district": "CENTRO",
            "postcode": "12345-678",
            "city": "CIDADE",
            "state": "RS"
        },
        "establishment": {
            "classification": "Convencional B1 Residencial",
            "supplyType": "Bifásico"
        }
    },
    "dates": {
        "actualReadingDate": "09/12/2021",
        "previousReadingDate": "08/11/2021",
        "nextReadingDate": "07/01/2022",
        "date": "12/21"
    },
    "debits": {
        "tusd": {
            "price": "21.4",
            "name": "Tarifa de Uso do Sistema de Distribuição",
            "abbreviation": "TUSD",
            "kWhAmount": "50.0"
        },
        "te": {
            "price": "16.99",
            "name": "Tarifa de Energia",
            "abbreviation": "TE",
            "kWhAmount": "50.0"
        },
        "cip": {
            "price": "1.17",
            "name": "Contribuição de Iluminação Pública",
            "abbreviation": "CIP"
        }
    },
    "powers": {
        "active": {
            "previousReading": "32182.0",
            "actualReading": "32202.0",
            "kWhAmount": "20.0"
        }
    },
    "installationCode": "0123456789",
    "price": "0.0",
    "realPrice": "56.08",
    "lastMonthPrice": "0.0"
}